src/libcore/convert.rs

```diff
@@ -350,7 +355,7 @@ pub trait Into<T>: Sized {
 /// [`String`]: ../../std/string/struct.String.html
 /// [`Into<U>`]: trait.Into.html
 /// [`from`]: trait.From.html#tymethod.from
-/// [book]: ../../book/ch09-00-error-handling.html
+/// [book]: ../../book/first-edition/error-handling.html
```

Comment:
I think the original link here is the correct one.  The first-edition is no longer included.

Reply:
Thanks for pinpointing this. I apparently have made an error, because it indeed seems like the removed line contained the correct reference. I will change it right away!